scaper/audio.py

```diff
@@ -1,18 +1,15 @@
 # CREATED: 4/23/17 15:37 by Justin Salamon <justin.salamon@nyu.edu>
 
-'''
+"""
 Utility functions for audio processing using FFMPEG (beyond sox). Based on:
```

Comment:
This is my bad, but we can update this top comment since we don't use ffmpeg anymore.

Reply:
cool, removed